tests/rptest/tests/rpk_topic_test.py

```diff
@@ -8,7 +8,6 @@
 # by the Apache License, Version 2.0
 
 from ducktape.utils.util import wait_until
-from ducktape.mark import ignore
```

Comment:
This test was already re-enabled in 3224ac19be3d398c674d3c17ac31d48ff768b48a -- maybe update the commit message to reflect that here we're just cleaning up the import.

Reply:
Fixed up!